homeassistant/helpers/service.py

```diff
@@ -21,6 +23,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+DESCRIPTION_CACHE = {}
```

Comment:
Don't store the description cache in a global. Instead store it in `hass.data`, which is a dict for globals during the runtime of a hass instance.